### ISSUE TITLE
Menu is missing (GNOME an XFCE)

### ISSUE THREAD

**Issue by ntr0py** [toxygen-project/toxygen#33]
Installing toxygen 0.2.6-12 on ubuntu xenial (16.04 LTS amd64). 
Trying to bootstrap new profile and add friends. Seems adding new friends is not obvious or even impossible.

How to reproduce:

Starting up from console (there is no .desktop launcher yet it seems).

Creating new profile, no password, dont store it in standard location produces first traceback (because it cant write in its program folder):

```
Traceback (most recent call last):
  File "/usr/bin/toxygen", line 9, in <module>
    load_entry_point('Toxygen==0.2.6.0', 'console_scripts', 'toxygen')()
  File "/usr/lib/python3.5/dist-packages/toxygen/main.py", line 486, in main
    toxygen.main()
  File "/usr/lib/python3.5/dist-packages/toxygen/main.py", line 141, in main
    ProfileHelper(path, name).save_profile(self.tox.get_savedata())
  File "/usr/lib/python3.5/dist-packages/toxygen/settings.py", line 240, in __init__
    os.makedirs(directory)
  File "/usr/lib/python3.5/os.py", line 241, in makedirs
    mkdir(name, mode)
PermissionError: [Errno 13] Keine Berechtigung: '/usr/lib/python3.5/dist-packages/toxygenavatars'
```

Alright, restarting and selecting to create new (empty) profile in the standard location, seems to work.

Starting with a clean profile it seems to be impossible to add new contacts. I tried right clicking on roster and expected to get a dialogue for adding a contacts TOX-ID. Nothing happens when starting with a new (empty) tox profile, but in the parent console I get a traceback:

```
Traceback (most recent call last):
  File "/usr/lib/python3.5/dist-packages/toxygen/mainscreen.py", line 548, in friend_right_click
    friend = Profile.get_instance().get_friend(num)
  File "/usr/lib/python3.5/dist-packages/toxygen/profile.py", line 152, in get_friend
    return self._contacts[num]
IndexError: list index out of range
```

Hmm maybe this is a bug and I would need to have a pre-populated (utox) profile?
Ok that loads, shows my contacts, but now how can I add another TOX-ID? Right clicking the roster is without any effect now, not even a traceback. Sorry if I miss something obvious here,
how can I add a contact?

Keep up the good work, im curious to see some progress :)

**Comment by ingvar1995** [comment]
Have you tried Profile => Add contact in menu? Btw, are you using packages fro tox.chat?

**Comment by ntr0py** [comment]
Thanks for your quick response.
Yes im using the packages from "deb https://pkg.tox.chat/debian nightly xenial". There are no menus to click onto (I thought the picture in the wiki is a mockup and toxygen is still in progress to there). 
So i could not try "Profile => Add":
![toxygen_xenial](https://cloud.githubusercontent.com/assets/843562/19879601/b535a58e-9ff4-11e6-91b2-8457505e724b.png)

**Comment by ingvar1995** [comment]
Saving to program directory fails because Toxygen doesn't have permission to write in /usr/lib/python3.5/ without sudo. If you want to save profile in this folder try to run Toxygen with sudo or allow writing to this directory using ``sudo chown``

About missing menu: I will try to fix it soon. Does Toxygen installed fom sources work in your case? Installation via package already installed all required deps, so you need just download code for test.

**Comment by ntr0py** [comment]
Yes I know that it can't write there, I just reported that because I thought it should not crash with a traceback: The user will not see that (if not started from terminal) and instead will get a report about toxygen crashed to submit to Canonical.

About the missing menu: yes im not sure why it will not diplay for me, I am using ubuntu-gnome (gnome-shell instead of unity) and i think it does not have those unity global menu mods installed, all other programs always show their menu properly (e.g. gajim).

A quick "git clone https://github.com/toxygen-project/toxygen.git" then "python3 toxygen/toxygen/main.py" does not display the menu for me (like in above picture from the .deb package).

I would be glad to help getting this to run properly on Ubuntu, so if you need any additional information let me know :)

**Comment by ingvar1995** [comment]
After 5477a7d Toxygen shows message box with info about error

About menu: can reproduce it on Ubuntu with GNOME (both PySide and PyQt4), seems to be one more bug in Qt on GNOME. I tried to fix it but without any success. Will try to fix it again later.

**Comment by ingvar1995** [comment]
Fixed in e854516
